### DISCUSSION TITLE
How to center the tabs in TabControl?

### DISCUSSION THREAD

**Discussion by ent3m** [AvaloniaUI/Avalonia#12223] [Q&A]
I know I can change the position of the tab panel with TabStripPlacement="Top/Bottom" and the direction with FlowDirection="LeftToRight/RightToLeft". However, I want my tabs to center horizontally instead of arranged from left to right or right to left. How can I do that?

**Comment by ent3m** [comment]
I figured it out.
`  <Style Selector="TabControl /template/ WrapPanel">
    <Setter Property="HorizontalAlignment" Value="Center"/>
  </Style>`

**Reply by joe-beach-supmfg** [reply]
Amazing. I kept trying to set the HorizontalAlignment of various elements to `Center`, but nothing was working. Saved me a lot of time.
